### ISSUE TITLE
Foreign key cannot be added manually

### ISSUE THREAD

**Issue by svenvarkel** [balderdashy/sails-mysql#191]
If DB tables are generated by sails-mysql the ID field's type is INT(10). When an association is created then the foreign key field type is INT(11). In order to create foreign keys in MySQL the data types of the primary key and the foreign key must match. 
Is there any reason why primary key is INT(10) and foreign key is INT(11)?

Update: it seems like the difference comes from one ID being SIGNED INT and the other UNSIGNED INT.

**Comment by devinivy** [comment]
Thanks for the PR– have you been using it this way successfully?  Mind writing a failing test for this issue?

**Comment by sailsbot** [comment]
Thanks for posting, @svenvarkel.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
